pkg/printer/workload_source_printer_test.go

```diff
@@ -183,3 +183,156 @@ branch:   my-branch
 		})
 	}
 }
+
```

Comment:
Add tests that have Ready condition true and asserts on `TIME` calculated since its `lastTransitionTime`